app/models/concerns/wp_seo.rb

```diff
@@ -0,0 +1,12 @@
+module WpSEO
+  extend ActiveSupport::Concern
+
+  included do
+    serialize :seo_fields
+  end
+
+  def update_wp_seo_attributes(json)
+    self.seo_fields = json['seo_fields']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.